source/developer/contribution-guide.md

```diff
@@ -66,6 +66,8 @@ After submitting a PR, before it is merged:
 3. Dev review must pass
     - Two core committers will review the pull request and either give feedback or approve the PR.
     - Any comments will need to be addressed before the pull request is ready to merge.
+4. CSS review must pass
```

Comment:
Thanks Jason, I've moved the CSS review to before the Dev review.

Re the contributing guidelines, I would say CSS review should happen after PM review and before Dev review.  Would we need a label in GitHub for that?